src/Provider/Photon/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "geocoder-php/photon-provider",
+    "type": "library",
+    "description": "Geocoder Photon adapter",
+    "keywords": [],
+    "homepage": "http://geocoder-php.org/Geocoder/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Jonathan Beliën"
+        }
+    ],
+    "require": {
+        "php": "^7.0",
```

Comment:
Why not 7.2?
That is the lowest active version

Reply:
I changed to PHP 7.2 but then I realized that Geocoder itself requires PHP 7.0 (see https://github.com/geocoder-php/Geocoder/blob/master/composer.json#L15).

Wouldn't it make more sense to keep PHP 7.0 as requirement for providers too ?

Reply:
Confirm by Travis : build is made with PHP 7.0 and 7.1 :)